packages/conf-flint/conf-flint.3.0/opam

```diff
@@ -22,12 +27,14 @@ depexts: [
   ["libflint-dev"] {os-distribution = "ubuntu" }
   ["flint"] {os = "macos" & os-distribution = "homebrew"}
   ["flint"] {os-distribution = "macports" & os = "macos"}
-  ["flint" "flint-devel"] {os-distribution = "centos"}
   ["flint" "flint-devel"] {os-family = "fedora"}
   ["flint" "flint-devel"] {os-distribution = "ol"}
-  ["flint-dev"] {os-distribution = "alpine"}
+  ["flint-dev@testing"] {os-distribution = "alpine"}
   ["flint-devel"] {os-family = "opensuse"}
-  ["libflint-devel"] {os = "win32" & os-distribution = "cygwinports"}
+  ["libflint-devel"] {os = "win32" &
+                         (os-distribution = "cygwinports" |
+                          os-distribution = "cygwin") }
+  ["mingw-w64-flint"] {os = "win32" & os-distribution = "msys2" }
```

Comment:
I can see your points:
- `flint` is [not available as a MinGW Cygwin package](https://cygwin.com/packages/package_list.html), so we instead just use the Cygwin package. Even though that may make CI happy (actually it still fails!) that's not the way to go, because the resulting executable will then require `cygwin1.dll` contrary to MinGW expectations
- the MSYS2 fix is fair enough. It is 64-bit only, but [MSYS2 doesn't offer a 32-bit package](https://packages.msys2.org/base/mingw-w64-flint)